frontend/src/citizen-frontend/calendar/ReservationModal.tsx

```diff
@@ -216,6 +217,9 @@ export default React.memo(function ReservationModal({
             )}
             hideErrorsBeforeTouched={!showAllErrors}
             data-qa="start-date"
+            onFocus={(ev) => {
+              scrollIntoViewSoftKeyboard(ev.target, 'start')
```

Comment:
Did you test these inputs with `'center'` on android emulator? Did the input still get stuck behind the keyboard? On my iphone, using `'start'` scrolls the view so far up that the background shows if using the simplest daily repetition. And `'center'` scrolls the input far enough above the keyboard on my android.

Reply:
`'center'` indeed is enough to get the input field to be visible, but with `'start'` almost all of the actual date picker is visible too (which would be a nice). I can't actually see any difference with `'start'` and `'center'` on my iPhone, but if `'center'` works okay on your device (instead of `'start'`), I'll remove this special case even though the date picker will not be completely visible anymore.

Reply:
I actually didn't think about the date picker itself. `'start'` is probably better then 👍